src/Microsoft.CSharp/src/Microsoft.CSharp.csproj

```diff
@@ -216,9 +220,33 @@
   </ItemGroup>
   <ItemGroup>
     <EmbeddedResource Include="Resources\$(AssemblyName).rd.xml" />
+    <EmbeddedResource Include="Resources\Strings.de.resx" />
+    <EmbeddedResource Include="Resources\Strings.es.resx" />
+    <EmbeddedResource Include="Resources\Strings.fr.resx" />
+    <EmbeddedResource Include="Resources\Strings.it.resx" />
+    <EmbeddedResource Include="Resources\Strings.ja.resx" />
+    <EmbeddedResource Include="Resources\Strings.ko.resx" />
+    <EmbeddedResource Include="Resources\Strings.ru.resx" />
+    <EmbeddedResource Include="Resources\Strings.zh-Hans.resx" />
+    <EmbeddedResource Include="Resources\Strings.zh-Hant.resx" />
```

Comment:
Do we expect all the libraries to have the same languages? If so we could consider adding these lists to the localization targets file to reduce the boilerplate more as well.

How did you chose the languages? I assume that list is subject to change?